pkg/util/azureclient/compute/virtualmachinescalesets.go

```diff
@@ -30,7 +30,6 @@ var _ VirtualMachineScaleSetsClient = &virtualMachineScaleSetsClient{}
 func NewVirtualMachineScaleSetsClient(ctx context.Context, log *logrus.Entry, subscriptionID string, authorizer autorest.Authorizer) VirtualMachineScaleSetsClient {
 	client := compute.NewVirtualMachineScaleSetsClient(subscriptionID)
 	azureclient.SetupClient(ctx, log, "compute.VirtualMachineScaleSetsClient", &client.Client, authorizer)
-	client.PollingDuration = 30 * time.Minute
```

Comment:
Why is this duration being removed? Does it have a sensible default we're overriding?

Reply:
15 minutes.  Previously I tried upping it to see if that would reduce our issues, but I don't think it did.